rust/unicode/src/lib.rs

```diff
@@ -13,9 +13,13 @@
 // limitations under the License.
 
 //! Unicode utilities useful for text editing, including a line breaking iterator.
+#![no_std]
+
+extern crate alloc;
+
 mod tables;
 
-use std::cmp::Ordering;
+use core::cmp::Ordering;
```

Comment:
I haven't done any `no_std` stuff before; is it standard to always use core, or do you use `std` when it's available? Do we want to have a feature? That's [what serde does](https://github.com/serde-rs/serde/blob/master/serde/src/lib.rs#L80) but I'm not sure about the rationale?

Reply:
AFAIK you just use core, since std just re-exports that anyway.

Reply:
sgtm!

Reply:
+1 jansol, `std` over simplified is just re-exported core + less portable additional features bundled together.